### DISCUSSION TITLE
git preview repo boxes status

### DISCUSSION THREAD

**Discussion by rahulkumar8987** [community/community#19469] [Other Feature Feedback, Questions, & Ideas]
hello community when i push repo then i check out my git box repo on  preview then no any kind of reflection i see, have you guys any solution please provide

**Comment by itsgareth** [comment]
inside of the directory can you run this command -> `git remote -v`

---

if you get this response -> `fatal: not a git repository (or any of the parent directories): .git`

- run -> `git init`
- run -> `git remote set-url origin <REPOSITORY_URL>`
**the repository URL can be found by going to your repo, selecting the green code button and copying the URL**

---
if you get this response -> `origin	https://github.com/USERNAME/REPO_NAME.git (fetch)`
then please make sure you're adding the files correctly.

- run `git status` to view all modified files
- run `git add <filename>` for each file you would like to commit
- run `git commit -m "your commit message"`
- run `git push origin main`
